certs/service.go

```diff
@@ -44,43 +55,56 @@ type Service interface {
 
 	// RevokeCert revokes a certificate for a given serial ID
 	RevokeCert(ctx context.Context, token, serialID string) (Revoke, error)
+
+	// Renew the exipred certificate from certs repo
+	RenewCerts(ctx context.Context, bsUpdateRenewCert bool) error
+
+	// Automattically trigger RenewCert function for given renew interval time
```

Comment:
Fix typo `Automatically` and add a comment for the next method.

Reply:
No more this function is needed in code. It was removed